b2sdk/urllib3/NOTICE

```diff
@@ -0,0 +1,8 @@
+Urllib3
+Copyright 2008-2011 Andrey Petrov and contributors.
+
+Copyright 2023 Backblaze Inc.
+Changes made to the original source:
+The following classes have been overriden for support HTTP 100-continue:
+urllib3.connection.HTTPConnection, urllib3.connection.VerifiedHTTPSConnection,
+urllib3.connectionpool.HTTPConnectionPool, urllib3.connectionpool.HTTPSConnectionPool
```

Comment:
these are not our (Backblaze) changes as you are suggesting thru the header above!

IMO we should just say we are vendoring part of botocore that is relavant to supporting 100 Continue and list our changes to it below it.